test/diff.js

```diff
@@ -21,6 +24,7 @@ void (
     [null, true],
     [null, ''],
     [null, 42],
+    [null, int(42)],
```

Comment:
Let's take this out of the array, and use the `diffsPrimitives` macro only if `BigInt` is available. We should make sure the title clarifies we're testing a BigInt primitive.

Reply:
I don't see this one appearing in the snapshot though.

Reply:
That sounds like a much better and less hack-ish way to handle this. I think you'll have a better idea of what that looks like? I just removed my changes to those files from my branch for now. (In case you want to do some deeper work on another branch.)